meeting_minutes/220915_SARIF_TC_66.md

```diff
@@ -0,0 +1,195 @@
+| Charles    | Wilson    |                   | Guest                   |
+| Chris      | Wysopal   | Veracode          | Member                  |
+| David      | Keaton    | Individual        | Chair                   |
+| Gerald     | Sullivan  | Micro Focus       | Voting Member           |
+| Jeff       | Williams  | Contrast Security | Voting Member           |
+| Johnathan  | Gilday    | Contrast Security | Member                  |
+| Mary       | Martin    | Microsoft         | Voting Member           |
+| Michael    | Fanning   | Microsoft         | Voting Member           |
+| Nathan     | Baird     | Microsoft         | Voting Member           |
+| Pavel      | Mikula    | Sonar             | Voting Member           |
+| Stefan     | Hagen     | Individual        | Secretary, taking notes |
+| Thanassis  | Avgerinos | ForAllSecure Inc  | Voting Member           |
+| Yekaterina | O'Neil    | Micro Focus       | Voting member           |
+
+## 1.3 Procedures for this meeting (Co-Chair David)
+
+## 1.4 Approval of agenda (Co-Chair David)
+
+* https://www.oasis-open.org/committees/download.php/70336/agenda_20220915.html
+
+Agenda was approved.
+
+## 1.5 Approval of previous minutes (Co-Chair David)
+
+* [Minutes of 2022-08-25_Meeting #65](https://www.oasis-open.org/committees/document.php?document_id=70335&wg_abbrev=sarif)
+
+Minutes were approved.
+
+## 1.6 Review of action items and resolutions (Secretary Stefan)
+
+* ACTION on David to keep pushing to close on the wikipedia page
+  * David kindly asks if someone can support
+  * ONGOING 
+* ACTION on Mary to continue to develop fixes scenario
+  * ONGOING 
+* ACTION on Michael to rename from master to main (URL)
+  * COMPLETE 
+* ACTION on Thanassis to present runtime-specific data (on September 15th)
+  * COMPLETE after this meeting's demo
+* ACTION on Michael to provide additional references for the wikipedia
+  * ONGOING
+* ACTION on Jeff to provide an initial seed page at github to foster the future roadmap / extension discussion
+  * Jeff kindly ask for advice on the location of such a page
+  * ONGOING
+* ACTION on Stefan to create a list for plausible potential as a github page for boosting the SBOM/other standards discussion
+  * COMPLETE per the minutes of 2022-08-11.
+
+## 1.7 Identification of SARIF TC voting members (Co-Chair David)
+
+### 1.7.1 Prospective members attending their first meeting
+
+### 1.7.2 Members attaining voting rights at the end of this meeting
+
+### 1.7.3 Members losing voting rights if they have not joined this meeting by the time it ends
+
+* Paul Anderson will lose voting status if he does not join before the end of this meeting
+### 1.7.4 Members who previously lost voting rights who are attending this meeting
+
+* Chris Wysopal will regain voting rights after the next meeting in case he joins it
+
+### 1.7.5 Members who have declared a leave of absence
+
+* Jeff Williams has declared a leave of absence until the first meeting in November
```

Comment:
This error was mine.  Jeff's leave of absence is just for the month of October; he plans to be at the remaining September meeting.

Reply:
The correction helps, but his leave of absence will not take effect until after the next meeting.  Someone reading the minutes in a vacuum might think he will not be allowed to attend the remaining September meeting.  How about "Jeff Williams intends to declare a leave of absence for the month of October."?

Reply:
Thanks. Noted intentions